### PR TITLE
D8UN-1977 & D8UN-1978

### PR DESCRIPTION
**Pull request by AdamMcFet**
Custom breadcrumb module for board pages & pathauto alias patterns configured for all content types.

### CODE REVIEW

**Review by DuttonMa — APPROVED** [review]
Nice work !

File: project/config/communityrelations/config/pathauto.pattern.board_member_pattern.yml [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
 dependencies:
   module:
     - node
-_core:
-  default_config_hash: IPBus0EzmucmooeZ0Uit-0IpbgDhfBhvfMgthjcZmRs
 id: board_member_pattern
 label: 'Board member pattern'
 type: 'canonical_entities:node'
-pattern: 'board-members/[node:title]'
+pattern: '[node:title]'
```
Comment (soda736) [inline project/config/communityrelations/config/pathauto.pattern.board_member_pattern.yml:10]: I think this pattern should be /about-us/board/[node:title]

File: project/sites/communityrelations/modules/custom/communityrelations_breadcrumbs/src/BoardBreadcrumb.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build(RouteMatchInterface $route_match) {
+    $breadcrumb = new Breadcrumb();
+    $title_resolver = $this->titleResolver->getTitle($this->request->getCurrentRequest(), $route_match->getRouteObject());
+    if ($this->node) {
+      $links[] = Link::createFromRoute(t('Home'), '<front>');
+      $links[] = Link::fromTextandUrl(t('Board'), Url::fromUri('internal:/about-board'));
```
Comment (soda736) [inline project/sites/communityrelations/modules/custom/communityrelations_breadcrumbs/src/BoardBreadcrumb.php:109]: This needs changed as the path to the board members page is https://www.community-relations.org.uk/board-members and not /about-board

**Review by soda736 — CHANGES_REQUESTED** [review]

File: project/config/communityrelations/config/pathauto.pattern.board_member_pattern.yml (hunk 1 above)
Comment (AdamMcFet) [reply project/config/communityrelations/config/pathauto.pattern.board_member_pattern.yml]: Done thank you Adam

File: project/sites/communityrelations/modules/custom/communityrelations_breadcrumbs/src/BoardBreadcrumb.php (hunk 2 above)
Comment (AdamMcFet) [reply project/sites/communityrelations/modules/custom/communityrelations_breadcrumbs/src/BoardBreadcrumb.php]: Good spot, thanks. Should be updated now.

**Review by soda736 — APPROVED** [review]

**Pull request merged**